.travis.yml

```diff
@@ -3,16 +3,22 @@ language: node_js
 node_js:
   - '0.10'
 
-env:
-  - TEST_SUITE=unit
-
 install:
   - npm install
-  - ./node_modules/.bin/mozilla-download ./firefox/ --branch=nightly --channel=prerelease
+  - ./node_modules/.bin/mozilla-download ./firefox/ --product firefox --branch mozilla-central
   - export FIREFOX_NIGHTLY_BIN="./firefox/firefox-bin"
```

Comment:
This must be changed to: `- export FIREFOX_NIGHTLY_BIN="./firefox/firefox/firefox-bin"`